wenet/bin/recognize.py

```diff
@@ -126,6 +129,19 @@
             assert len(arr) == 2
             char_dict[int(arr[1])] = arr[0]
     eos = len(char_dict) - 1
+    #Load HLG  for wfst-based decoding
+    if args.mode == 'wfst_based_decoding':
+        d = torch.load('data/graph/HLG.pt')
```

Comment:
it's better to add how HLG was built.

Reply:
Yes, I will finish it in the near future